python/api/serializers.py

```diff
@@ -778,6 +782,27 @@ def get_content(self, obj):
         return obj.content
 
 
+class LogStorageSerializer(serializers.Serializer):
+    name = serializers.CharField(read_only=True)
+    type = serializers.CharField(read_only=True)
+    format = serializers.CharField(read_only=True)
+    body = serializers.SerializerMethodField()
+
+    def get_body(self, obj):
+        body = obj.body
+
+        if body is None and obj.name == 'ansible':
+            path_file = os.path.join(
+                config.RUN_DIR, f'{obj.job.id}', f'{obj.name}-{obj.type}.{obj.format}')
+            with open(path_file, 'r') as f:
+                body = f.read()
+
```

Comment:
Here we can return check log interactively: if body is None get data from CheckLog table